code/modules/power/supermatter/supermatter.dm

```diff
@@ -1046,8 +1046,10 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			zap_str = 0
 		//This gotdamn variable is a boomer and keeps giving me problems
 		var/turf/T = get_turf(target)
-		var/pressure = max(1,T.return_air().return_pressure())
-		//We get our range with the strength of the zap and the pressure, the lower the former and the higher the latter the better
+		var/pressure = 1
```

Comment:
```suggestion
		var/pressure = max(1,T?.return_air()?.return_pressure()) || 1
```

Reply:
alternatively
```
var/pressure = T?.return_air()?.return_pressure())
pressure = max(1, pressure)
```

Reply:
or even 
```
var/pressure = get_turf(target)?.return_air()?.return_pressure()
pressure = max(1, pressure)
```

Reply:
I respect you spooky but I refuse to maul my code with ?. I want it to be as understandable as possible and they just don't make sense for that goal to me.

Reply:
Isn't this a reasonably hot proc? 
Using ?. Is faster since it's one operation instead of two

Reply:
Max(1,null) is probably 1 so you can at least reduce that complexity

Reply:
It is a reasonably hot proc but I'm not sure if the time saved is worth the readability.
If I had to use any of these it would probably be `max(1,T?.return_air()?.return_pressure())`, although this line from the reference gives me pause. 

> If ?. is used after a proc call, a list lookup, or a complex expression where the type can't be known, it will act like ?: instead.

Reply:
This is a microopt, so unless you're going to prove it's a significant win, the readability is more important.